.vscode/.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
```

Comment:
[OPTIONAL]
By default, the `.vscode` folder only contains the `settings.json` file which doesn't need to be pushed to GitHub. Kindly remove the `.github/workflows` folder as you already have it and add the `.vscode` folder to gitignore 👍